libs/api/domains/driving-license/src/lib/graphql/models/qualityPhoto.model.ts

```diff
@@ -1,10 +1,16 @@
-import { Field, ObjectType } from '@nestjs/graphql'
+import { Field, ObjectType, ID } from '@nestjs/graphql'
 
 @ObjectType()
 export class QualityPhoto {
+  @Field(() => ID)
```

Comment:
I wonder if we can remove this decorator but keep the field for your field resolver. 

Since it feels like internal logic, should always match the authenticated user, and not relevant to the client.